### PR TITLE
Feat/OIDC

### PR DESCRIPTION
**Pull request by royscheeren**
# Scope

Add POC and demo to use OIDC with SIWT

# Work done

- Updated siwt.xyz with a demo
- add OIDC client package 
- add OIDC bridge package
- add ory hydra as oidc provider

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,76 @@
-# SIWT
+# Siwt
 
-Sign In With Tezos (SIWT) is a library that supports the development of your decentralized application (dApp) by:
+<a alt="Nx logo" href="https://nx.dev" target="_blank" rel="noreferrer"><img src="https://raw.githubusercontent.com/nrwl/nx/master/images/nx-logo.png" width="45"></a>
 
-- proving the users ownership of the private key to the address the user signs in with,
```
Comment (jdsika) [inline README.md]: The override of the readme seems unintentional?

File: package.json [hunk 2]
```diff
@@ -2,45 +2,47 @@
   "name": "siwt",
   "version": "0.0.6",
```
Comment (jdsika) [inline package.json]: I guess this should be 0.0.7 but we need to bump to 0.0.8 anyway for the release

**Comment by jdsika** [comment]
Why did you delete the LICENSE file?

File: README.md [hunk 3]
```diff
@@ -8,5 +8,3 @@ Sign In With Tezos (SIWT) is a library that supports the development of your dec
 - and TZIP-33 Sign-In with Tezos (SIWT).
 
 After winning the [Tezos x Encode hackathon](https://medium.com/encode-club/encode-x-tezos-hackathon-finale-prizewinners-and-summary-ba5c9cda677c) and due to the positive feedback from the Tezos community, we decided to spend additional efforts to take our Proof-of-Concept to an easy-to-integrate library with an extended feature set with the help of the [Tezos Foundation](https://tezos.foundation/). We invite you to use SIWT in your dApp and contact us in our [Discord](https://discord.com/invite/6J3bjhkpxm?utm_source=StakeNow+Discord&utm_medium=Twitter&utm_campaign=StakeNow.Fi+Launch) if you have any questions! You can read about the contribution guidelines [here](https://github.com/StakeNow/.github/blob/main/README.md).
-
-> **Note** Get started with the [online documentation](https://docs.siwt.xyz/).
```
Comment (jdsika) [inline README.md]: Why delete the link to the documentation?

File: package.json [hunk 4]
```diff
@@ -1,46 +1,48 @@
 {
   "name": "siwt",
-  "version": "0.0.6",
+  "version": "0.0.7",
```
Comment (jdsika) [inline package.json:3]: Are we introducing a new feature that would need a version bump? Then please put v0.0.8 everywhere and we make a release after this PR

Comment (royscheeren) [reply package.json:3]: No, as I mentioned 😂

File: packages/discord-bot-ui/src/app/common/signIn/signIn.ts [hunk 5]
```diff
@@ -4,7 +4,7 @@
  * SPDX-License-Identifier: MIT
  */
 import { DAppClient, NetworkType, RequestSignPayloadInput } from '@airgap/beacon-sdk'
-import { useSiwt } from '@siwt/react'
```
Comment (jdsika) [inline packages/discord-bot-ui/src/app/common/signIn/signIn.ts:7]: Is the react package now deprecated and are you deleting it?

File: packages/discord-bot-ui/vite.config.ts [hunk 6]
```diff
@@ -0,0 +1,35 @@
+/// <reference types='vitest' />
+import { nxViteTsPaths } from '@nx/vite/plugins/nx-tsconfig-paths.plugin'
```
Comment (jdsika) [inline packages/discord-bot-ui/vite.config.ts:2]: I see a lot of changes in the discord bot. Is that actually working again?

File: package.json (hunk 4 above)
Comment (jdsika) [reply package.json:3]: JUST wanted to be extra sure ;)

File: packages/docs.siwt.xyz/package.json [hunk 7]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "docs-siwt-xyz",
-  "version": "0.0.6",
```
Comment (jdsika) [inline packages/docs.siwt.xyz/package.json:3]: Pretty sure this must be 0.0.7 ?

File: packages/discord-bot-ui/vite.config.ts (hunk 6 above)
Comment (royscheeren) [reply packages/discord-bot-ui/vite.config.ts:2]: Most of the changes are around tooling, version updates and formatting. It needs some more work to be brought back up to date.

File: packages/docs.siwt.xyz/static/img/oidc.siwt.png
Comment (jdsika) [inline packages/docs.siwt.xyz/static/img/oidc.siwt.png:1]: Love it!

File: packages/discord-bot-ui/src/app/common/signIn/signIn.ts (hunk 5 above)
Comment (royscheeren) [reply packages/discord-bot-ui/src/app/common/signIn/signIn.ts:7]: Not officially, but it should be.

**Review by jdsika — DISMISSED** [review]
I approve as far as I can check it :D

**Pull request merged**